backend/backend/utils/auth.py

```diff
@@ -85,7 +85,7 @@ async def login(db_session: AsyncSession, user: UserSchema) -> str:
                 pool,
                 bcrypt.verify,
                 user.password,
-                db_user.password_hash
+                db_user.password_xhash
```

Comment:
You probably did this by accident:
```suggestion
                db_user.password_hash
```